app.rb

```diff
@@ -0,0 +1,127 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def list_people
+    if @people.empty?
+      puts 'There are no people in the library'
+    else
+      @people.each do |person|
+        if person.instance_of?(Student)
+          puts "[Student]: Name: #{person.name}, Id: #{person.id}, Age: #{person.age}"
+        else
+          puts "[Teacher]: Name: #{person.name}, Id: #{person.id}, Age: #{person.age}"
+        end
+      end
+    end
+  end
+
+  def create_person
```

Comment:
- [ ] Good job so far :+1:  I'm able to create and list the book I created. But when I tried to create a person  (teacher/student) I'm getting errors. The same thing to adding rentals. Please make sure that all your method and parameters like parent_permission are well spelled and well implemented in order to make your app works as expected :+1: 

#### Errors when I tried to add a teacher 
![errors(1)](https://user-images.githubusercontent.com/6615394/211155746-04ad298e-978c-468c-97cc-3bbc46778a57.jpg)